redactive-ai/redactive
# chore: moved to hatch build system

### What problem to solve?
To build python package with a better build system and introduce automated release Github Action
### How to implement?
- Updated config for [hatch](https://github.com/pypa/hatch)
- Added CI for publishing python SDK to test.pypi.org for testing
### How to test?
- Run `pre-commit run --all` to trigger all the checking
- Add release tag with newer version with this branch (feel free to tag, I will delete after merge)
- Check https://github.com/redactive-ai/redactive/actions/workflows/python-sdk.yaml
- Check https://test.pypi.org/project/redactive/
